### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by Kingvigusson** [BrianLima/UWPHook#86] [Q&A]
I am no expert or computer genius, but I cannot install UWPHook, SharpSteam, or the other program needed for visual studios. Every time I load the project, build and run I get loads of error messages saying NuGet packages are not installed and to use NuGet package manager and install or reinstall them, but does not say what packages are missing. I have all .NET framework files including 4.6.2 but can not get them to work. is there something I am doing wrong? can someone be able to precisely explain what I need to do as the instructions on how to install are not clear how to install any of the programs that are needed?

**Comment by BrianLima** [comment]
Do you want to code in the project or to use the program? 
If you only want to use UWPHook you neet to download either the .exe installer or the zip version from the downloads page, install or unzip the downloaded file and then follow the instructions:

https://github.com/BrianLima/UWPHook/releases/

**Reply by Kingvigusson** [reply]
thank you i did not see that bit really confused on why I couldn't get it to work hahaha
